SEN/README.md

```diff
@@ -14,18 +14,20 @@ To respect TTN standards, please check the [TTN documentation](../TTN/README.md)
 #include <MKRWAN.h>
 #include <DHT.h>
 
-#define APP_EUI "xxxxxxxxxxxxxxxx"
-#define APP_KEY "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
+#define APP_EUI "0000000000000000"
+#define APP_KEY "AC6DC50997CF1761BC19C1E32D8E9CCE"

```

Comment:
Try not to expose this information on git.
First, it might be a security breach.
Second, it's specific to the device, thus, someone reusing the script should be aware that he or she has to complete this by hand.